siriuspy/siriuspy/dvfimgproc/csdev.py

```diff
@@ -63,9 +63,9 @@ def _create_database(self):
         database['ImgProperties-Cte'] = database['Properties-Cte']
         del database['Properties-Cte']
 
-        # TODO: Version here will take value from siriuspy package
-        # Is this what we want? maybe we should start composing the string
-        # from siriuspy + machine-applications?
+        # TODO: Version here will take value from siriuspy package, when
+        # code is moved to this repo. Is this what we want? maybe we should
```

Comment:
the construction of this phase is weird. maybe consider re-phrasing it.

Reply:
done!